Makefile

```diff
@@ -77,7 +76,7 @@ test:
 
 .PHONY: e2e
 e2e: $(call GETENVOY_OUT_PATH,$(GOOS),$(GOARCH))
-	go test $(E2E_OPTS) $(E2E_EXTRA_OPTS) $(E2E_PKG_LIST)
+	(cd test/e2e && go test $(E2E_OPTS) $(E2E_EXTRA_OPTS))
```

Comment:
e2e now use a separate go.mod file

Reply:
what's the rationale for this? (For VSCode, this breaks the completion). I would be surprised if this reduces the size of getenvoy binary

Reply:
this is to make sure the dependencies are split and we don't accidentally add dependencies. Remember we started this because people were sloppy and get envoy was something like 50MB

Also, e2e deps shouldn't bleed or skew versions. What completion is broken?

Reply:
another way is to not add deps just for e2e.. I can also do that (basically it only needs untar I think...)

Reply:
if it is Makefile completion is breaking on, a better solution would be to add makefile to e2e. First step need to understand what is breaking.

Reply:
I will replace this with a big README instead as it is simpler. As long as we are careful, sharing one go.mod is ok!